axelarnetwork/axelar-core
# feat(tss): queries for active old keys

## Description

We should be able to retrieve the old rotated keys that are still active. This is particularly important to allow validators to figure out which keys they need to recover their shares from. To this end, this PR introduces the tss queries `old-active-keys` and `old-active-keys-by-validator`. The former enables us to retrieve the these keys by chain & role, whereas the later allows us to obtain the the old active keys that some validator has shares from.

## Todos


## Steps to Test

Just generate and rotate a few keys and then use the queries to check that they can be obtained.

## Expected Behaviour

## Other Notes
